### ISSUE TITLE
Constant updates in BlackBerry World

### ISSUE THREAD

**Issue by achan** [EOLed/snooby#99]
http://www.reddit.com/r/SnoobyApp/comments/1chs1u/does_anyone_else_have_this_problem_with_snooby/

> I've been talking to BlackBerry about this issue for about a month... They've been a little slow in responding... and whenever they do respond they tell me it's fixed and essentially close my ticket. (They've "fixed" it 3 times!!!) When I respond telling them it's not fixed, I get an auto-response email telling me to include my ticket number (as if the ticket number I already included is no longer valid).
> 
> Then I start a new ticket.... and the cycle continues.
> 
> The problem is this:
> 
> When I first submitted my app, there was a version number in a configuration file... I assumed the version number was for BlackBerry so I kept it the same as in documents: http://developer.blackberry.com/html5/documentation/code_sample_creating_a_bb_widget_config_document_834683_11.html
> 
> As you can see, the version number there is 2.0.0.0. I had originally thought it was the version number for the XML schema maintained by BlackBerry... so I left it as 2.0.0.0 and submitted my app to BlackBerry World.
> 
> I did this for v1.0 and v1.0.1.  
> 
> v1.0.2 rolls around and I realize the errors of my ways and [change my version number to 1.0.2](https://github.com/achan/snooby/commit/054bd82a8076254335436313622c9e158e6feddc#L1L4) so that it would match the correct version number. However, it turns out that BlackBerry World has cached the version number.... so now you have v1.1.3 from BlackBerry World and they think v2.0.0.0 is available and telling you to download it.
> 
> The quick fix is to up my version number to 2.1... but I'd prefer to have it properly fixed by BlackBerry seeing as how Snooby is not ACTUALLY at v2.1.
> 
> It was my mistake for putting 2.0.0.0 as the initial version... but BlackBerry really should provide a way for me to undo my wrongs...
> 
> **TLDR: The version number was incorrectly set to 2.0.0.0 and now that I've correctly set my version number back to 1.1.3, it thinks you should download 2.0.0.0.**
> 
> I am very sorry for this. I've received tons of flak (understandably) for this and I'm hoping it'll be resolved asap. I'm planning a design of the app soon which will be version 2.0. If BlackBerry hasn't gotten back to me about this yet by then (we're still in talks), I will version it as 2.1 instead.

PM [/u/Deathstar31](http://www.reddit.com/user/Deathstar31) when resolved.
